urdf_model/include/urdf_model/utils.h

```diff
@@ -0,0 +1,67 @@
+/*********************************************************************
+* Software License Agreement (BSD License)
+*
+*  Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
Ok, I replaced Willow Garage with OSRF in b14f20e47033338f2b7c4dca5c9a978ae8a521a1. Is that ok?